src/ThorFIP.bonsai

```diff
@@ -4438,6 +4439,7 @@
               <Expression xsi:type="VisualizerMapping" />
               <Expression xsi:type="VisualizerMapping" />
               <Expression xsi:type="viz:TableLayoutPanelBuilder">
+                <viz:Name>FIP Camera</viz:Name>
```

Comment:
Just "Behavior Cameras" (w/BoxNum) would be better?
Now that all Bonsai instances are headless anyway, RA don't need to know that Cameras are visualized/recorded by FIP bonsai.

Reply:
Thats true, I should remove the "FIP" part. 

If the FIP boxes will always be the "D" box, then I can label it as "Camera D", which would match the format of the other rigs.

Reply:
The FIP box will not always be the D box, for example in 428.

I think "FIP Camera" is the most clear, since in 447 the RAs need to open the "C" camera manually, but not the FIP boxes.